packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -6443,79 +6443,79 @@ Expected request body:
 {}
 ```
 
-### Type_Property_Optional_Plaindate_getAll
+### Type_Property_Optional_PlainDate_getAll
 
-- Endpoint: `get /type/property/optional/plaindate/all`
+- Endpoint: `get /type/property/optional/plainDate/all`
 
 Expected response body:
 
 ```json
 { "property": "2022-12-12" }
 ```
 
-### Type_Property_Optional_Plaindate_getDefault
+### Type_Property_Optional_PlainDate_getDefault
 
-- Endpoint: `get /type/property/optional/plaindate/default`
+- Endpoint: `get /type/property/optional/plainDate/default`
 
 Expected response body:
 
 ```json
 {}
 ```
 
-### Type_Property_Optional_Plaindate_putAll
+### Type_Property_Optional_PlainDate_putAll
 
-- Endpoint: `put /type/property/optional/plaindate/all`
+- Endpoint: `put /type/property/optional/plainDate/all`
 
 Expected request body:
 
 ```json
 { "property": "2022-12-12" }
 ```
 
-### Type_Property_Optional_Plaindate_putDefault
+### Type_Property_Optional_PlainDate_putDefault
 
-- Endpoint: `put /type/property/optional/plaindate/default`
+- Endpoint: `put /type/property/optional/plainDate/default`
 
 Expected request body:
 
 ```json
 {}
 ```
 
-### Type_Property_Optional_Plaintime_getAll
+### Type_Property_Optional_PlainTime_getAll
 
-- Endpoint: `get /type/property/optional/plaintime/all`
+- Endpoint: `get /type/property/optional/plainTime/all`
 
 Expected response body:
 
 ```json
 { "property": "13:06:12" }
 ```
 
-### Type_Property_Optional_Plaintime_getDefault
+### Type_Property_Optional_PlainTime_getDefault
 
-- Endpoint: `get /type/property/optional/plaintime/default`
+- Endpoint: `get /type/property/optional/plainTime/default`
```

Comment:
just wondering if we have some convention for path, i notice some paths use `-` as break.

Reply:
both is acceptable for azure

https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#http-url-casing